changelog/wave/v1.10.3.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Wave v1.10.3
+date: 2024-08-05
+tags: [wave]
+---
+
+## Feature updates and improvements
+
+- Added remove from redis in invalidateAuthorization by @munishchouhan in https://github.com/seqeralabs/wave/pull/591
```

Comment:
Is "Remove from redis" an action or a config? Different fonts might make this clearer.

Reply:
Something in between. I've updated accordingly.